docs/semgrep-assistant/customize.md

```diff
@@ -0,0 +1,206 @@
+   2. **Include a notification in the PR/MR comment**: Show developers likely false positive findings in PR/MR comments, but include a note explaining why Assistant thinks the finding may be safe to ignore.
+
+Findings filtered out by Assistant can be reviewed at any time in Semgrep by going to the [**Code > Pre-production backlog** page](https://semgrep.dev/orgs/-/findings?tab=open&last_opened=All+time&backlog=preprod). Semgrep also allows you to agree with the filtering to close the finding or disagree to reopen.
+
+## Add Memories (beta)
+
+Assistant Memories allows admins to tailor Assistant's remediation guidance to their organization's standards and defaults on a per-project, per-rule basis. You can provide feedback by adding custom instructions whenever Assistant gives a suggested fix.
+
+Memories are enabled by default for all organizations with Assistant enabled.
+
+### Add a memory based on Assistant's suggested fix
+
+To add a memory modifying a suggested fix presented by Assistant:
+
+1. Identify the specific instance of **Assistant's suggested fix** that you want to modify. These can be found on the finding details page or in the PR or MR comment.
+2. Click **Customize fix** to open an input box, and enter your preferred remediation approaches and secure defaults for the project. Your suggestion can be as general as "Use AWS Secrets Manager to manage secrets."
+   ![Assistant’s suggested fix for a hardcoded secret in the user’s code](/img/memories-3.png#md-width)
+   ***Figure***. Assistant’s suggested fix for a hardcoded secret in the user’s code.
+3. Click **Save and regenerate**.
+   ![User-provided custom instruction that Assistant uses to improve future guidance.](/img/memories-2.png#md-width)
+   ***Figure***. User-provided custom instruction that Assistant uses to improve future guidance.
+4. Assistant regenerates the suggested fix to reflect the instructions you provided.
+   ![Regenerated Assistant fix using the user-provided instructions.](/img/memories-1.png#md-width)
+   ***Figure***. Regenerated Assistant fix using the user-provided instructions.
+
+While Assistant Memories is in **public beta**, memories are scoped to remediation guidance on a per-project and per-rule basis. A saved memory only affects future guidance for findings triggered by the same rule in the same project.
+
+### Add memory during triage
+
+If you identify findings that are safe to ignore and write triage notes indicating why this is so, Assistant can store this information as a memory and use it when assessing whether a similar finding should be shown to developers in the future. Assistant also takes that memory, reanalyzes similar findings in your backlog, and suggests issues that may be safe to close.
```

Comment:
```suggestion
When you identify findings that are safe to ignore, Semgrep Assistant can store your triage notes and create a **memory**, which it will then use to assess if a similar finding should be shown to developers. Assistant also takes that memory, reanalyzes similar findings in your backlog, and suggests issues that may be safe to close.
```
Something like that?

Reply:
LGTM!